ci/packaging/suse/rpmfiles_maker.sh

```diff
@@ -1,40 +1,81 @@
 #!/bin/bash
+##########################################
+# generate skuba tar.gz and specfile
+##########################################
 
+##########################################
+# Housekeeping
 set -e
-
-susepkg_dir=$(cd "$( dirname "$0" )" && pwd)
 tmp_dir=$(mktemp -d -t skuba_XXXX)
-rpm_files="${susepkg_dir}/obs_files"
-devel_prj="Devel:CaaSP:5"
-version="$1"
-tag="$2"
-closest_tag="$3"
-
 log()   { (>&2 echo ">>> $*") ; }
 clean() { log "Cleaning temporary directory ${tmp_dir}"; rm -rf "${tmp_dir}"; }
+trap clean ERR
+susepkg_dir=$(cd "$( dirname "$0" )" && pwd)
+rpm_files="${susepkg_dir}/obs_files"
+devel_prj="Devel:CaaSP:5"
 
-if [ $# -ne 3 ]; then
+##########################################
+# CLI args
+if (( $# != 3 )); then
     log "usage: <version> <tag> <closest-tag>"
     exit 1
 fi
+version="$1"     # pretty version string for display 
+tag="$2"         # can be ugly refs/tags/xxx or specific commit hash
+closest_tag="$3" # usually $tag, but might be commit
 
-trap clean ERR
+##########################################
+# env vars which come from GitHub Actions
+TARBALL_PATH=${TARBALL_PATH:-${tmp_dir}/skuba.tar.gz}
+TEMPLATE_PATH=${TEMPLATE_PATH:-${susepkg_dir}/skuba_spec_template}
+SPECFILE_PATH=${SPECFILE_PATH:-${tmp_dir}/skuba.spec}
+
+##########################################
+# other global stuff
+
+function main {
+    gen_tar
+    gen_spec
+    if [[ "${GITHUB_ACTIONS:-}" != "true" ]]
+    then
+        # GitHub Actions calls this script directly
+        make CHANGES="${tmp_dir}/skuba.changes.append" suse-changelog
+
+        # Assemble for OBS / manual use later 
+        rm -rf "${rpm_files}"
+        mkdir -p "${rpm_files}"
+        cp "${tmp_dir}"/* "${rpm_files}"
+        log "Find files for RPM package in ${rpm_files}"
+
```

Comment:
I left the local execution essentially unchanged, so it generates a directory instead of a tarball.  The only addition was to also generate the body for the GitHub release comment, making copy-paste a little easier.